apps/api/src/controllers/routes/newsletter.py

```
@@ -298,3 +303,200 @@ def get_campaign_stats(campaign_id):
+            to_email=email,
+            subject=subject,
+            html_body=html_body,
+            plain_body=plain_body
+        )
+
+        if success:
+            logger.info(f"Email de verificação enviado para {email}")
+        else:
+            logger.warning(f"Falha ao enviar email de verificação para {email}")
+
+        return success
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)